### ISSUE TITLE
Can't load font via options.fontSrcPath

### ISSUE THREAD

**Issue by Yu-AnChen** [chunyenHuang/hummusRecipe#11]
https://github.com/chunyenHuang/hummusRecipe/blob/56b917611c403e491949c230e8275149ffd6e758/lib/Recipe.js#L40-L41

At this point, `options` will be `{}`. Tried to use `this.options.fontSrcPath` instead of `options.fontSrcPath`, but got this `Segmentation fault: 11` in the console.

**Comment by chunyenHuang** [comment]
Do you pass the font file path or the parent dir path?

This function should be rename to `fontSrcDirPath` because it will load all the font files in the dir path.

**Comment by Yu-AnChen** [comment]
https://github.com/chunyenHuang/hummusRecipe/blob/56b917611c403e491949c230e8275149ffd6e758/lib/Recipe.js#L40-L41

I passed the path of the directory that has all the font files. 

When you call `this._setParameters(options = {}); ` at line 40, the `options = {}` will be viewed as a new variable instead of the `options` in the initial Recipe call `new Recipe(src, output options)`. So I changed line 40 to `this._setParameters(options)`, I can log out and see that the fonts are registered, but still getting a `Segmentation fault: 11`

```
{ '.ds_store': '/Users/yu-an.chen/projects/symposium/server/tools/makeBadge/fonts/.DS_Store',
  'notosanscjktc-light': '/Users/yu-an.chen/projects/symposium/server/tools/makeBadge/fonts/NotoSansCJKtc-Light.ttf',
  'notosanscjktc-medium': '/Users/yu-an.chen/projects/symposium/server/tools/makeBadge/fonts/NotoSansCJKtc-Medium.ttf',
  'roboto-regular': '/Users/yu-an.chen/projects/symposium/server/tools/makeBadge/fonts/Roboto-Regular.ttf',
  roboto: '/Users/yu-an.chen/projects/symposium/server/tools/makeBadge/fonts/Roboto.ttf' }
Segmentation fault: 11
```

**Comment by chunyenHuang** [comment]
Thanks for reporting. This is fixed in version `1.3.6`.

**Comment by Yu-AnChen** [comment]
It's working now, thanks!
